src/pages/settings/PreferencesPage.js

```diff
@@ -38,42 +48,74 @@ const priorityModes = {
     },
 };
 
-const PreferencesPage = ({priorityMode}) => (
-    <>
-        <HeaderGap />
-        <HeaderWithCloseButton
-            title="Preferences"
-            shouldShowBackButton
-            onBackButtonPress={() => redirect(ROUTES.SETTINGS)}
-            onCloseButtonPress={() => redirect(ROUTES.HOME)}
-        />
-        <View style={styles.pageWrapper}>
-            <View style={[styles.settingsPageBody, styles.mb6]}>
-                <Text style={[styles.formLabel]} numberOfLines={1}>
-                    Priority Mode
-                </Text>
-                <View style={[styles.mb2]}>
-                    {/* empty object in placeholder below to prevent default */}
-                    {/* placeholder from appearing as a selection option. */}
-                    <RNPickerSelect
-                        onValueChange={
-                            mode => NameValuePair.set(CONST.NVP.PRIORITY_MODE, mode, ONYXKEYS.PRIORITY_MODE)
-                        }
-                        items={Object.values(priorityModes)}
-                        style={styles.picker}
-                        useNativeAndroidPickerStyle={false}
-                        placeholder={{}}
-                        value={priorityMode}
-                        Icon={() => <Icon src={DownArrow} />}
-                    />
+class PreferencesPage extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            isNewsStatusEnabled: !(this.props.user.expensifyNewsStatus === false),
+        };
+        this.toggleNewsStatus = this.toggleNewsStatus.bind(this);
+    }
+
+    toggleNewsStatus(toggle) {
```

Comment:
We typically prefer a param that describes the state since they can be a bit more clear as to what the boolean is doing. With `toggle`, if it's `false` does that mean don't toggle or set it to `false`? By using `isSubscribed` then it's a bit more clear that true/false pertain to if the user is subscribed.
```suggestion
    toggleNewsStatus(isSubscribed) {
```